lib/openstax/aws/stack.rb

```diff
@@ -179,7 +191,8 @@ def apply_change_set(params: {}, wait: false)
         template_url: template.s3_url,
         capabilities: capabilities,
         parameters: self.class.format_hash_as_stack_parameters(params),
-        change_set_name: "#{name}-#{Time.now.utc.strftime("%Y%m%d-%H%M%S")}"
+        change_set_name: "#{name}-#{Time.now.utc.strftime("%Y%m%d-%H%M%S")}",
```

Comment:
not necessary here, but we might consider moving time stuff to a time module sometime...to encapsulate formatting stuff like this (so that just this stuff can be isolated unit tested and then not worry about it working in the code)